marketplace-service/src/main/java/com/axonivy/market/repository/ProductJsonContentRepository.java

```diff
@@ -7,5 +7,5 @@
 @Repository
 public interface ProductJsonContentRepository extends MongoRepository<ProductJsonContent, String> {
 
-  ProductJsonContent findByProductIdAndVersion(String productId, String version);
+  ProductJsonContent findByProductIdAndVersion(String productId, String tag);
```

Comment:
this one is danger, because we are search by set of fields, then maybe a list will be return